taxcalc/utils.py

```diff
@@ -406,7 +411,26 @@ def results(c):
                 outputs.append(getattr(c.records, col))
         else:
             outputs.append(getattr(c, col))
-    return DataFrame(data=np.column_stack(outputs), columns=STATS_COLUMNS)
+
+    res = DataFrame(data=np.column_stack(outputs), columns=STATS_COLUMNS)
+
+    # Keep itemized deductions (c04470) where itemized deductions reduce AGI
```

Comment:
On Fri, 10 Jul 2015, T.J. Alumbaugh wrote:

> 
> In taxcalc/utils.py:
> 
> > @@ -406,7 +411,26 @@ def results(c):
> >                  outputs.append(getattr(c.records, col))
> >          else:
> >              outputs.append(getattr(c, col))
> > -    return DataFrame(data=np.column_stack(outputs), columns=STATS_COLUMNS
> )
> > +
> > +    res = DataFrame(data=np.column_stack(outputs), columns=STATS_COLUMNS)
> > +
> > +    # Keep itemized deductions (c04470) where itemized deductions reduce 
> AGI

The comment should read "keep itemized deductions where they reduce 
taxable income" (not AGI).

dan

> 
> Hmm.. this function had previously served the purpose of "collect up the
> columns we care about and return them." Now it will do some useful work. Is
> there a better place for this logic? Totally possible that the answer is
> 'no', or 'maybe, but we can figure it out after merging'. This PR has been
> up for a while, apologies for just raising this now.
> 
> —
> Reply to this email directly or view it on
> GitHub.[AHvQVfKSzOejFY5d0AazwbVqfPio-tGEks5ob9tUgaJpZM4Eny74.gif]
> 
> 
>